pkg/tfbridge/schema.go

```diff
@@ -1076,7 +1075,7 @@ func MakeTerraformOutputs(
 
 		// Next perform a translation of the value accordingly.
 		out := MakeTerraformOutput(ctx, p, value, tfi, psi, assets, supportsSecrets)
-		//if !out.IsNull() {
+		// if !out.IsNull() {
```

Comment:
Maybe an opportunity to delete this? Looks like dead code?

Reply:
Yeah, good point, thanks for flagging it.